configuration.rst

```diff
@@ -953,6 +953,12 @@ namespace ``Symfony\Config``::
         $security->accessControl(['path' => '^/admin', 'roles' => 'ROLE_ADMIN']);
     };
 
+.. caution::
+
+    Note that nested configs (e.g. ``\Symfony\Config\Framework\CacheConfig``) cannot be used for
```

Comment:
Could we emphasise more on what we **can** do instead of **can't**.

```
Note that only use "root" ConfigBuilder are supported (e.g. ``\Symfony\Config\FrameworkConfig``). Other "nested" builders (e.g. ``\Symfony\Config\Framework\CacheConfig``) cannot be autowired...

Reply:
I agree with you that we should emphasize more what we can do, rather than what we can't. But you're not quite right that nested configs are builders: it not builders, it POPOs.

I have tried to rewrite notice to emphasize that fact.